Description: tsc workarounds
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2023-12-13

--- a/build.ts
+++ b/build.ts
@@ -20,6 +20,7 @@
       console.info('Done! Watching...');
       let debounce: NodeJS.Timer;
       chokidar.watch(path.join(__dirname, 'src'), { ignored: '*.json', ignoreInitial: true }).on('all', () => {
+// @ts-ignore
         clearTimeout(debounce);
         debounce = setTimeout(
           () =>
